packages/common/src/jsonApiClient/utilities/shouldModifyRelationship.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function shouldModifyRelationship({
+  resourcePath,
+  relationshipsToModify = [],
+  relationKey,
+}) {
+  if (relationshipsToModify.includes('all')) {
+    return true
+  }
+  const relationshipPath = `${resourcePath}.${relationKey}`
+  return relationshipsToModify.some(str => {
```

Comment:
cool I didn't know about `some`. I've been using `find` for that kind of thing, this is neat